### PR TITLE
Add moth week as a holiday

### PR DESCRIPTION
**Pull request by Mothblocks**
:cl:
add: NanoTrasen now officially recognizes Moth Week as a holiday.
/:cl:

Completely untested. If you're going to merge it merge it quick since it ends in 4 days!

[why]: # http://nationalmothweek.org/

Not sure about other servers but Bagil is having fun with moth week (I think).

### CODE REVIEW

**Comment by Jared-Fogle** [comment]
No fucking way, it broke? Now I'll have to actually run it.

**Comment by 81Denton** [comment]
How aboot adding some station prefixes that include moth species?
```
/datum/holiday/moth/getStationPrefix()
	return pick("Mothball","Lepidopteran","Lightbulb","Moth","Giant Atlas","Twin-spotted Sphynx","Madagascan Sunset","Luna","Death's Head","Emperor Gum","Polyphenus","Oleander Hawk","Io","Rosy Maple","Cecropia","Noctuidae","Giant Leopard","Dysphania Militaris","Garden Tiger")
```

**Comment by Jared-Fogle** [comment]
I wanted to but couldn't think of anything. Thanks for the list.

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -366,6 +366,16 @@
 	begin_month = JUNE
 	begin_weekday = SUNDAY
 
+/datum/holiday/moth
+	name = "Moth Week"
+	begin_day = 21
+	begin_month = JULY
+	end_day = 29
```
Comment (Cyberboss) [inline code/modules/holiday/holidays.dm]: It says during the last full week of July, so these shouldn't be hardcoded days.

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/holiday/holidays.dm (hunk 1 above)
Comment (Jared-Fogle) [reply code/modules/holiday/holidays.dm]: I have been embarrassed on my own domain (fixing)

**Comment by optimumtact** [comment]
maybe next year

**Comment by Jared-Fogle** [comment]
not all is lost as there is still thankfully today and tomorrow

**Pull request merged**